### DISCUSSION TITLE
Release Version - White screen appears after the splashscreen (only release version, filename.aab) (Cordova 12.0.0)

### DISCUSSION THREAD

**Discussion by leonelgregioauler** [apache/cordova#422] [Q&A]
I have a problem.

I ran the command: (I use Oracle JET, 14.1)
ojet build android --release --buildconfig="C:/PUBLISH/NEOCP3001/build-android.json"

The release generates, I upload it to Google Play, I create an internal test version, I download it, I install it on the cell phone, but after the splashscreen is displayed, the screen turns white.

The key was generated using the command:
keytool -genkey -v -keystore neocp3001.jks -keyalg RSA -keysize 2048 -validity 10000 -alias neocp3001

Debug mode installs and runs perfectly!

Cordova 12.0.0
Target SDK 33
SDK Build Tools 33.0.2
gradle 7.6
Android Gradle Plugin 7.4.2
JAVA 19

![image](https://github.com/apache/cordova/assets/12154804/af0fe331-81e7-451a-8b75-fb447df2187d)

![SplashScreen](https://github.com/apache/cordova/assets/12154804/fec3fa66-eb66-48df-b263-2c7281df4e56)

![Screen with error](https://github.com/apache/cordova/assets/12154804/cd2eabab-7595-40b1-a60b-afbedc3c1a14)

**Comment by breautek** [comment]
There is very little information to go on, what pointers that we can give is going to be rather limited.

A white screen is likely caused by your index page either not loading at all, or erroring on load. 

Normally the web inspector is disabled on release builds, but you can enable it by setting the `InspectableWebview` preference to `true`. Additionally you can test release builds locally by using `cordova run android --release -- --packageType=apk`. Pay special attention to the double hyphen! (Windows powershell I think requires quotes: `cordova run android --release "--" --packageType=apk`). Don't forget to remove the preference before you make an actual release, otherwise end users might be able to load up the web inspector for your app.

Generally speaking debug builds should work identically to release builds as far as the native cordova shell goes, but release mode has stricter policies for SSL encryptions whereas debug builds often ignore SSL errors to accept self-signed certificates. Loading up the web inspector should give you insight on what is actually happening though.

**Comment by leonelgregioauler** [comment]
Good morning !

I used the preference below, and performed the debug, finding the error. It's really an error in Oracle JET, I need to fix it. 

I didn't know how to debug the release version, this will help me a lot !!
Thanks !!

preference name="InspectableWebview" value="true"

![image](https://github.com/apache/cordova/assets/12154804/0f59a6b0-e4bf-41f4-a8c7-f04ff9cdc159)
